tests/queries/tests.py

```diff
@@ -3800,3 +3800,77 @@ def test_ticket_23622(self):
             Ticket23605A.objects.filter(qx),
             [a2], lambda x: x
         )
+
+
+class ExcludeCombineTests(TestCase):
```

Comment:
I'd like to make a decision about this separately. I agree that the queries/tests.py file is hard to maintain, so we need to do something here.